core/src/internal/scrim/scrim.mod.resolver.ts

```diff
@@ -159,6 +160,7 @@ export class ScrimModuleResolver {
         @Args("leaveAfter", {type: () => Int}) leaveAfter: number,
         @Args("group", {nullable: true}) groupKey?: string,
         @Args("createGroup", {nullable: true}) createGroup?: boolean,
+        @Args("canSaveDemos", {type: () => Boolean, nullable: true}) canSaveDemos?: boolean,
```

Comment:
Why is this an arg to the mutation? Shouldn't we get this info from the implicitly player making the request?

Reply:
Unsure. Ticket said to add a checkbox for the player that they can save it that will only show up if they can save replays. Use case might be they have accounts that can save demos, but they're playing on one that can't. @0c370t